src/pages/Main/Main.js

```diff
@@ -1,7 +1,104 @@
-import React from 'react';
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import ShopCarousel from './components/ShopCarousel/ShopCarousel';
+import SortBar from './components/SortBar/SortBar';
+import ShopList from './components/ShopList/ShopList';
+
+import { MainNav } from '../../components/MainNav/MainNav';
 
 const Main = () => {
-  return <div>메인 페이지</div>;
+  const [shopList, setShopList] = useState([]);
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/shop.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => setShopList(data));
```

Comment:
```suggestion
    fetch('/data/shop.json')
      .then(res => res.json())
      .then(data => setShopList(data));
```
fetch함수의 get method는 method의 default 값입니다. 따라서 생략이 가능합니다. 
또한 http://localhost:3000도 생략 가능합니다.